calico-enterprise/index.md

```diff
@@ -1,135 +1,62 @@
-      <td>SIEM integrations</td>
-      <td style="text-align: center">&nbsp;</td>
-      <td style="text-align: center">✓</td>
-    </tr>
-    <tr class="table-header">
-      <th colspan="3">Observability</th>
-    </tr>
-    <tr>
-      <td>Dynamic service graph</td>
-      <td style="text-align: center">&nbsp;</td>
-      <td style="text-align: center">✓</td>
-    </tr>
-    <tr>
-      <td>Application Layer observability</td>
-      <td style="text-align: center">&nbsp;</td>
-      <td style="text-align: center">✓</td>
-    </tr>
-    <tr>
-      <td>Dynamic packet capture</td>
-      <td style="text-align: center">&nbsp;</td>
-      <td style="text-align: center">✓</td>
-    </tr>
-    <tr>
-      <td>DNS dashboards</td>
-      <td style="text-align: center">&nbsp;</td>
-      <td style="text-align: center">✓</td>
-    </tr>
-    <tr>
-      <td>Flow visualizer</td>
-      <td style="text-align: center">&nbsp;</td>
-      <td style="text-align: center">✓</td>
-    </tr>
-    <tr class="table-header">
-      <th colspan="3">Management</th>
-    </tr>
-    <tr>
-      <td>Multi cluster, multi-cloud & hybrid cloud networking</td>
-      <td style="text-align: center">&nbsp;</td>
-      <td style="text-align: center">✓</td>
-    </tr>
-  </tbody>
-</table>
+
+![calico-cloud]({{site.baseurl}}/images/calico-cloud-small.png)
+
+## You can solve these 3 problems today...with Calico Cloud
+
+- **Workload access controls**
+
+    Implement pod-level workload access controls and protect containerized environments in the Kubernetes cluster from outside threats, while enabling applications and workloads to securely communicate with resources outside the cluster behind a firewall or other control point.
+
+- **Compliance**
+
+    Create compliance policies for any compliance framework including PCI, SOC 2, and more. Continuously monitor compliance for your container, Kubernetes, and cloud environment. Retain a daily history of your compliance status. Generate on-demand or scheduled audit reports.
+
+- **Faster troubleshooting**
+
+    Enable faster troubleshooting of Kubernetes workloads and applications with Service Graph, packet capture, anomaly detection, and performance hotspots, leading to shorter time-to-resolution, less application downtime, and improved quality of service.
+
+#### Don't have a {{site.prodname}} cluster? Easy...
+
+{% include open-new-window.html text='Try it now' url='https://www.tigera.io/tigera-products/cloud-trial' %}
+
+#### Already have a {{site.prodname}} cluster? You are **5 minutes** away from connecting to Calico Cloud. 
+
+What happens when you connect/disconnect your cluster to Calico Cloud? See {% include open-new-window.html text='Connect your cluster to Calico Cloud' url='https://docs.calicocloud.io/get-started/connect-cluster' %}.
+
+1. Identify the cluster you want to use and apply the following manifest.
+
+   You can use any {{site.prodname}} operator-installed cluster that is configure with `kubectl`.
```

Comment:
I think there's a typo in this sentence where it should say `configured` instead of `configure`.

However, even with that typo fixed, I don't think it makes sense to say "you can use a cluster configured with kubectl".

I think you're trying to say two things:
1. Any cluster
2. Kubectl must be connected to it.

My suggestion:
```suggestion
   You can use any {{site.prodname}} operator-installed cluster. Ensure `kubectl` is connected, and then execute the following:
```